specs/devexperience/DevExperience.md

```diff
@@ -0,0 +1,181 @@
+
+With that being said, one thing that will be supported is users having a mix of old UWP and newer Reunion projects in the same .sln file or repository. This will allow developers to migrate slowly at their own pace. In order to do this, the task assemblies that ship as part of Reunion must be renamed. Otherwise, MSBuild will mistakenly use the wrong assemblies.
+
+### 5.1.4 – Helping Developers call the right APIs
+Many WinRT APIs do not work in certain scenarios. Whether they don’t work because of the difference in App model, or whether they require Package Identity, we will help developers where possible, to make the right API calls in their application. We will generate a list of API calls that don’t work in Win32, as well as APIs that require Package identity, and provide tooling for .NET developers through a Roslyn Analyzer.
+Here is the list of APIs that we can use as a starting point: [Windows Runtime APIs available to a packaged desktop app (Desktop Bridge)](https://docs.microsoft.com/en-us/windows/apps/desktop/modernize/desktop-to-uwp-supported-api)
+
+Prior Art: We started on a Roslyn Analyzer for [incompatible WinUI3 APIs](https://github.com/dotnet/try-convert/blob/feature/winui/src/WinUI.Analyzer/Analyzer/DeprecatedUseAnalyzer.cs).
+
+#### 5.1.4.1 – Analyzer for App Authors
+With the WindowsPackageType property, we can expose the exact type of application the customer is building, and therefore provide the exact type of information that they need.
+
+#### 5.1.4.2 – Analyzer for Library authors
+Since library authors won’t set the WindowsPackageType property, the analyzer for library authors should be general purpose, and provide messages such as “this API behaves differently in Desktop vs UWP, add a runtime check or use <replacement API>”.
+
+### 5.2 – New Apps and WinUI3
+
+#### 5.2.1 - New WinUI3 Apps
+Creating new WinUI3 apps will have the proper manifest files included, and project files by design. Developers will use Visual Studio to create new projects, and .NET developers will have the options of using the .NET CLI. There will be a separate doc more specifically focused on the new project experience for Windows developers that will go into more detail.
+
+##### 5.2.1.1 - Manifest Files
+All new WinUI3 apps will have a package.appxmanifest file, but the SxS manifest should be removed. The tooling will supply default manifest files, which it can use mt.exe to merge. The “default” manifests include:
```

Comment:
> ...the SxS manifest should be removed.

What SxS manifest are you referring to?

Reply:
There is a default one in new WinUI3 projects